prometheus_ricoh_printer_exporter/main.py

```diff
@@ -2,96 +2,57 @@
 # Oskar Druska 2022
 # For further information look up LICENSE.txt
 
-# Exporter entry point; handles URLs and BeautifulSoup objects
+# Exporter entry point
 
-import sys
-import os
 import argparse
 import time
-import requests
 import logging
-import csv
-from bs4 import BeautifulSoup as bs
+import urllib.parse
 from prometheus_client import start_http_server, REGISTRY
-from . import exporter_file
+from .data import get_urls
+from . import exporter
+
+DEFAULT_HOSTNAME = '0.0.0.0'
```

Comment:
I assume this is the interface that the exporter listens on, to provide the prometheus metrics (rather than a default target printer). If so, I recommend renaming this to `DEFAULT_LISTEN_INTERFACE`, as it's more explicit.